src/components/graphs/pagination/Pagination.less

```diff
@@ -107,7 +94,6 @@
 }
 
 .link-container{
```

Comment:
if this selector is no longer being used, maybe we can delete it all? Doesn't look like it has content anymore

Reply:
I deleted all of the empty selectors already. I think you might be looking at an individual commit.

Reply:
ok got it